recipe/meta.yaml

```diff
@@ -43,56 +45,45 @@ requirements:
         - sed                                    # [unix]
         - python                                 # [build_platform != target_platform]
         - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+        - qt6-main                               # [build_platform != target_platform]
+        - doxygen
     host:
         - libboost-devel
         - python
-        - qt-main
-        - qt-webengine
+        - qt6-main
+        - pyside6
         - occt
+        - smesh
         - xerces-c
         - zlib
         - vtk
         - eigen
         - pybind11
         - coin3d
-        - pyside2
-        - smesh
-        - doxygen  # [build_platform == target_platform]
         - hdf5
         - freetype
         - libspnav  # [linux]
         - pcl
         - six
         - ply
         - matplotlib-base
+        - yaml-cpp
+        - pivy
     run:
-        - pyside2
-        - vtk
-        - {{ pin_compatible('occt', max_pin='x.x.x') }}
-        - qt
-        - xerces-c
-        - zlib
-        - hdf5
         - python
-        - coin3d
+        - pyside6
         - pivy
-        - {{ pin_compatible('smesh', max_pin='x.x.x') }}
-        - freetype
         - numpy
         - six
         - pyyaml
-        - libspnav  # [linux]
-        - pcl
         - ply
-        - gmsh
         - graphviz
-    run_constrained:
-        - __osx >={{ MACOSX_DEPLOYMENT_TARGET|default("11.0") }}   # [osx and arm64]
-        - __osx >={{ MACOSX_DEPLOYMENT_TARGET|default("10.13") }}  # [osx and x86_64]
+        # - gmsh
+        # - libspnav  # [linux]
 
 test:
     commands:
-        - freecadcmd -t 0  # [unix and build_platform == target_platform]
+        # - freecadcmd -t 0  # [unix and build_platform == target_platform]
```

Comment:
What's stopping us from re-enabling these tests?

Reply:
Yes, sure these should be re-enabled.